app/controllers/sessions_controller.rb

```diff
@@ -37,4 +25,32 @@ def destroy
     flash[:success] = "You have logged out."
     redirect_to '/'
   end
+
+  private
+
+  def redirect_logged_in(user)
+    if user.regular?
+      redirect_to '/profile'
+    elsif user.merchant_employee?
+      redirect_to '/merchant/dashboard'
+    elsif user.admin?
+      redirect_to '/admin/dashboard'
+    end
+  end
+
+  def regular_user_logged_in?
```

Comment:
These Boolean conditions can be moved into the parent class as helper methods.
They can be re used in the nav bar view.